server/src/entity/Restaurant.ts

```diff
@@ -101,6 +104,30 @@ export class Restaurant extends BaseEntity {
   })
   public isActive: boolean;
 
+  @Column({
+    default: false,
+  })
+  public hasWifi: boolean;
+
+  @Column({
+    default: false,
+  })
+  public hasTV: boolean;
+
+  @Column({
+    default: false,
+  })
+  public hasParking: boolean;
+
   @OneToMany(() => Menu, (menu) => menu.restaurant)
   public menus: Menu[];
+
+  @OneToMany(() => Review, (review) => review.restaurant)
```

Comment:
Invalid syntax `@ManyToOne(type => User, user => user.photos)`

Reply:
What do you mean by invalid syntax... it works